src/index.md

```diff
@@ -29,7 +30,7 @@ srcset="assets/300w/logo_Unfold.jl-dark.png",alt="Unfold.jl Logo dark"
    alt="Unfold.jl Logo bright"
    class="h-[45vh] object-cover"          
  />
-</picture></a></td></tr> 
+</picture></a></td><td>B. Ehinger</td></tr> 
```

Comment:
mh. I'm not sure about this one. Let's discuss on Wednesday.

If we take it, I would rather put "Maintainer" there, but I think this clutters more than helps